barretenberg/cpp/src/barretenberg/vm/avm_trace/avm_execution.cpp

```diff
@@ -23,6 +24,8 @@
 
 using namespace bb;
 
+extern std::string avm_dump_trace;
```

Comment:
It would help readability to have suffix like "path" or "file" in the name of this variable.